### DISCUSSION TITLE
Splitter如何实现类似vscode的效果

### DISCUSSION THREAD

**Discussion by fpsqdb** [ant-design/ant-design#50956] [Q&A]
比如分隔线active样式、改变光标、双击恢复默认值
类似 https://www.npmjs.com/package/allotment

https://github.com/user-attachments/assets/a8fb856a-56a0-4258-9beb-d8215df18552

**Comment by wanpan11** [comment]
目前没有支持这个🤧，没法通过代码达到这个效果
